### DISCUSSION TITLE
Нерф SWAT и Cебов ( Или как их там на ЦДБГ )

### DISCUSSION THREAD

**Discussion by Kyper1** [octothorp-team/public-dobrograd#706] [Предложения]
Заявляю от лица всех краймеров!

Данные персонажи очень имбовые. Чрезмерное закидывание людей гранатами, ладно если бы они еще нормально работали, а не цепляли тебя даже тогда, когда ты в укрытии. Дымы, находящиеся по ту сторону стены, заставляют тебя кашлять, что очень сильно бесит. Снизьте им броню хотя бы до 70%, а то даже с дробовика в плотную убить их сложно. Добавьте хотя бы возможность скрафтить этот противогаз для ст. токаря за большие деньги, 100.000Р-150.000Р. В общем: Человек, который точно в свободное время может тренировать свой аим на механике  сервера+100 Брони+ возможность непрерывного закидывания кривых гранат и в среднем их 3. Три мать его терминатора. И даже, если ты как-то чудом с ними справишься, то тебя ждут еще серые и желтые лычки снаружи, которые добьют тебя, как скота.

**Comment by Kyper1** [comment]
Те, кто стоит на стороне защиты этих персонажей - игроки, которые мейнят профу полицейского.

**Reply by Black-White228337** [reply]
Ну вообще если представить реальных сотрудников SWAT и SEB, они вроде не снаряжаются по легкому как попало, у них даже шлемы есть, и то что ты их при штурме убить не можешь это понятно, у них есть хоть какие то тактики в отличие от рядовых краймеров которые просто встали за укрытия и стоят там с глоками, ну и так-то Доброград это не просто пострелушки, там ведь еще и РП есть, и ведь может проблема в тебе, а не в сервере? Некоторые люди и с обычными глоками могут Сватовцов перестрелять потому что они умеют стрелять может быть? И типа твое заявление что это защищает только те кто берут копов, то я  могу сказать что это поддержит только тот кто стрелять не умеет.

**Reply by Kyper1** [reply]
Дружище, я же написал, два выстрела из xm вплотную в торс они могут выдержать. Это невозможно.

**Comment by gasw1n** [comment]
Нормальный краймер до спецназа не доведёт. Если вы натворили дел что обычная полиция справиться не в силах, идёт спецназ-Логично. Делай выводы сам, никто нерфить спецназ не будет. 
by ex swat, op seb.

**Reply by Kyper1** [reply]
Это зависит от стечений обстоятельств. Не всегда получается быстренько скрыться или не попасться в руки полицейским в ходе погони.

**Reply by Kyper1** [reply]
И вообще есть краймеры, которые краймят не ради выгоды, а из-за самого процесса и не всегда интересно, просто ограбив кого-то, сразу прятаться, менять одежду и сидеть ждать, пока тебя не закончат разыскивать. Я и состою в их числе, мне иногда хочется устроить нормальную перестрелку, включая: снайперские и автоматические винтовки, дробовики. Серые и жёлтые лычки с таким вооружением вообще не ощущаются. Но сват, которого ты фиг убьёшь из-за гребанных гранат, просто надоедает.
